src/styles/blocks/stars-star.css

```diff
@@ -0,0 +1,9 @@
+.stars__star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: 50%;
+  justify-content: space-between;
+  margin-right: 4px;
```

Comment:
please for this margin use selector :not(:last-child) to remove it from last star of each row